### DISCUSSION TITLE
Editor mode in diagrams

### DISCUSSION THREAD

**Discussion by LHumblot** [dbeaver/dbeaver#35898] [General]
Hello every body i have an Academic license of the Enterprise Edition. It provides a full access to the product. But i can't manage to access editor mode in my diagrams. Must i activate something ? 
Can you help me to find a solution ?
Thanks for all.
Lionel

**Comment by LonwoLonwo** [comment]
Hello @LHumblot 

How exactly it doesn't work? Could you please provide a screen from the ERD window and the DBeaver version?

![изображение](https://github.com/user-attachments/assets/07819370-048e-46d2-be0d-ebf11c2651d2)
